### ISSUE TITLE
Improve upload progress

### ISSUE THREAD

**Issue by MeikelLP** [qcasey/paperless_share#8]
Instead of popping up a message every time a file is done uploading, a upload progress in the notifications would be awesome.

Alternatively you could show progress and a history in the app.

**Comment by qcasey** [comment]
Yeah, I like this. You mention history, which Android 11 does system-wide. I'd love to see file upload history in there.
